geometry/cbox.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include <vector>
+
+namespace m2
+{
+class CBox
```

Comment:
Please provide a short description of this class.

Reply:
Done